### PR TITLE
Correct the navigation

### PR DESCRIPTION
**Pull request by przemekjuzwiszin**

### CODE REVIEW

File: awaria.html [hunk 1]
```diff
@@ -6,12 +6,21 @@
...
     <header class="header">
-        <a class="header__link" href="index.html">Strona główna</a>
-        <a class="header__link" href="awaria.html">Awaria</a>
+        <nav class="navigation">
+            <ul class="navigation__list">
+                <li class="navigation__item">
+                    <a class="navigation__link" href="index.html">Strona główna</a>
+                </li>
+                <li class="naviation__link">
+        <a class="navigation__link" href="awaria.html">Awaria</a>
```
Comment (robson60000) [inline awaria.html:20]: The indentation looks a little weird on this line. The same in other files

File: awaria.html [hunk 2]
```diff
@@ -6,12 +6,21 @@
...
 <body>
     <header class="header">
-        <a class="header__link" href="index.html">Strona główna</a>
-        <a class="header__link" href="awaria.html">Awaria</a>
+        <nav class="navigation">
+            <ul class="navigation__list">
+                <li class="navigation__item">
+                    <a class="navigation__link" href="index.html">Strona główna</a>
+                </li>
+                <li class="naviation__link">
```
[the remark this answers is not in the document]
Comment (przemekjuzwiszin) [reply awaria.html:19]: It's a mistake, I'm correcting.

File: awaria.html (hunk 1 above)
Comment (przemekjuzwiszin) [reply awaria.html:20]: I'll do document formatting which should help.

File: terminal-pod.html [hunk 3]
```diff
@@ -6,12 +6,21 @@
...
+        <nav class="navigation">
+            <ul class="navigation__list">
+                <li class="navigation__item">
+                    <a class="navigation__link" href="index.html">Strona główna</a>
+                </li>
+                <li class="naviation__link">
+        <a class="navigation__link" href="terminal-pod.html">Terminal</a>
+                </li>
+            </ul>
+        </nav>
```
[the remark this answers is not in the document]
Comment (przemekjuzwiszin) [reply terminal-pod.html:23]: I think I made a mistake while replacing the text. Next time I will make a correction and check the change more closely.

File: awaria.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (przemekjuzwiszin) [reply awaria.html:19]: done

File: terminal-pod.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (przemekjuzwiszin) [reply terminal-pod.html:23]: done

File: awaria.html (hunk 1 above)
Comment (przemekjuzwiszin) [reply awaria.html:20]: And done.

**Review by przemekjuzwiszin — COMMENTED** [review]
I checked It looks good this time.

File: jak-dziala-pod.html [hunk 4]
```diff
@@ -6,12 +6,21 @@
...
 <body>
     <header class="header">
-        <a class="header__link" href="index.html">Strona główna</a>
-        <a class="header__link" href="jak-dziala-pod.html">Jak działa</a>
+        <nav class="navigation">
+            <ul class="navigation__list">
+                <li class="navigation__item">
+                    <a class="navigation__link" href="index.html">Strona główna</a>
+                </li>
+                <li class="naviation__link">
```
Comment (robson60000) [inline jak-dziala-pod.html:19]: navigation__link -> navigation__item

File: internet-pod.html [hunk 5]
```diff
@@ -6,12 +6,21 @@
...
 <body>
     <header class="header">
-        <a class="header__link" href="index.html">Strona główna</a>
-        <a class="header__link" href="internet-pod.html">Internet</a>
+        <nav class="navigation">
+            <ul class="navigation__list">
+                <li class="navigation__item">
+                    <a class="navigation__link" href="index.html">Strona główna</a>
+                </li>
+                <li class="naviation__link">
```
Comment (robson60000) [inline internet-pod.html:19]: navigation__link -> navigation__item

**Review by karbowskam — COMMENTED** [review]
I'm sorry for my previous comment, I forgot, that we need to talk with each other in english language :) 
But here you are 52 changes! Don't do that for us! ;) In my opinion, It's too much for one pull request. You added normalize, you fixed typo etc. - I think these could be in separate requests.

**Review by karbowskam — APPROVED** [review]

**Comment by przemekjuzwiszin** [comment]
You have absolutely right. Next time I do more branch and add more pull  @karbowskam

**Pull request merged**